cypress-tests/cypress/jsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "include": ["./node_modules/cypress", "cypress/**/*.js"],
+  "include": ["**/*", "cypress/**/*.js"],
```

Comment:
```suggestion
  "include": ["./**/*", "cypress/**/*.js"],
```

Reply:
Getting no input were found in config file error, if I made the above changes.

Reply:
Added the changes.